vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/ElementSteps.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.bdd.mobileapp.steps;
+
+import static org.apache.commons.lang3.Validate.isTrue;
+
+import java.util.Map;
+
+import org.jbehave.core.annotations.When;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.remote.RemoteWebElement;
+import org.vividus.bdd.mobileapp.model.PickerWheelDirection;
+import org.vividus.bdd.monitor.TakeScreenshotOnFailure;
+import org.vividus.bdd.steps.ui.validation.IBaseValidations;
+import org.vividus.selenium.IWebDriverProvider;
+import org.vividus.selenium.WebDriverUtil;
+import org.vividus.selenium.manager.IGenericWebDriverManager;
+import org.vividus.ui.action.search.Locator;
+
+@TakeScreenshotOnFailure
+public class ElementSteps
+{
+    private final IBaseValidations baseValidations;
```

Comment:
`baseValidations` should go last in the list of fields and in the constructor parameters

Reply:
hmm what is our code convention regarding to constructor param sequence?

Reply:
we don't have, but you can improve it :) obviously `validations` field is used after `webDriverProvider` so  `validations` should go after `webDriverProvider` and etc.

Reply:
following the usage sequence it will be hard to track the sequence of class dependencies and their usage once we have a normal class with more than one method and few dependencies. I think we can for example put dependencies in alphabet order

Reply:
put into alpha order

Reply:
alphabetical order doesn't make sense in such cases, for example: would you put `password` before `username` in some class storing credentials?